main.go

```diff
@@ -55,6 +57,14 @@ func (c *BasicPlugin) Run(cliConnection plugin.CliConnection, args []string) {
 		}
 		fmt.Println(org.Entity.Name + "/" + space.Entity.Name + "/" + app.Entity.Name)
 	}
+
+	if *target {
+		_, err = apps[0].Target(cliConnection)
```

Comment:
Is it possible to bind a route to multiple apps in different spaces?  When I tried I received: ```Server error, status code: 400, error code: 210003, message: The host is taken: doesthiswork
```  but perhaps there's another way of doing it that I'm unaware.